Sources/DIWise/DIConfigurator.swift

```diff
@@ -6,12 +6,9 @@
 //
 
 import Foundation
-import Segment
-import UIKit
-import CoreLocation
 
 @objcMembers
-public open class DIConfigurator: NSObject {
+public class DIConfigurator: NSObject {
```

Comment:
why remove `open` here? We need to be able to extend this.

Reply:
oh my bad, I was getting an error for a duplicate modifier here. It looks as though the compiler is annoyed that we use both `public` and `open`. I think we can remove `public` then since it's similar but more restricted